grammar/es/src/rules.rs

```diff
@@ -813,6 +1402,26 @@ pub fn rules_temperature(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()
                      latent: false,
                  })
              });
+    b.rule_2("<temp> Kelvin",
+             temperature_check!(),
+             b.reg(r#"k(?:elvin)?"#)?,
+             |a, _| {
+                 Ok(TemperatureValue {
+                     value: a.value().value,
+                     unit: Some("kelvin"),
+                     latent: false,
+                 })
+             });
+    b.rule_2("<temp> Kelvin",
+             number_check!(),
+             b.reg(r#"kelvin"#)?,
+             |a, _| {
+                 Ok(TemperatureValue {
```

Comment:
Why this rule on top of the preceding one?

Reply:
One is for phrasing with "degrees" and the other one without